.github/workflows/_unittest.yml

```diff
@@ -14,7 +14,7 @@ on:
 
 jobs:
   linux:
-    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@release/0.2
```

Comment:
It seems like this has to be a pytorch release branch, e.g. release/2.3, in this case.

I'm checking what vision does: https://github.com/pytorch/vision/blob/release/0.18/.github/workflows/docs.yml#L17
However, their script is quite misleading, it's using 2.1 everywhere: https://github.com/pytorch/vision/blob/main/packaging/cut_release.sh#L4-L23

Reply:
Oh test-infra is doing branch cut in sync with pytorch, so you want to use `@release/2.3` here.  On the other hand, this is a small issue for ET and I think it's ok to use `@main` too.  But let's stick with `@release/2.3`